### PR TITLE
SEO-176113-Core-Missing-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the missing H1

Regards,
Beryl.

### CODE REVIEW

File: aspnet-core/RadialMenu/TemplateSupport.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 layout: post
-title: Syncfusion Radial Menu Template Support.
-description: template support.
+title: Template Support | ASP.NET Core Radial Menu | Syncfusion
```
Comment (Yvone-Atieno) [inline aspnet-core/RadialMenu/TemplateSupport.md:3]: Template Support in ASP.NET Core Radial Menu Control | Syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet-core/RadialMenu/TemplateSupport.md (hunk 1 above)
Comment (berylouma) [reply aspnet-core/RadialMenu/TemplateSupport.md:3]: Added

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**